gdx/src/com/badlogic/gdx/graphics/ImmutableColorCpy.java

```diff
@@ -0,0 +1,124 @@
+
+package com.badlogic.gdx.graphics;
+
+/** A color class that prevents mutation to r/g/b/a values. */
+public class ImmutableColorCpy extends Color {
```

Comment:
The docs here should at least mention that this creates mutable copies of itself (with the `Color` class) when modified.